interactive-resume.js

```diff
@@ -0,0 +1,8 @@
+function disapearingAct() {
+  let email = document.getElementById('contactemail')
+  onclick = email.classList.toggle('show')
```

Comment:
you don't need to make email.classList.toggle() equal anything. it can just be it's own statement